File: beginner/1071.py
x = int(input())
y = int(input())
soma = 0

if x < y:
	x += 1
	while x < y:
		if x % 2 == 1:
			soma += x 
		x += 1
		
if x > y:
	y += 1
	while y < x:
		if y % 2 == 1:
			soma += y 
		y += 1

		
print(soma)

File: beginner/1071.c
# include <stdio.h>

int main(){
	int x, y, soma;
	scanf("%i", &x);
	scanf("%i", &y);
	soma = 0;
	
	if(x < y){
		x += 1;
		while(x < y){
			if(x % 2 == 1 || x % 2 == -1)
				soma += x; 
			x += 1;
		}
	}
	
	if(x > y){
		y += 1;
		while(y < x){
			if(y % 2 == 1 || y % 2 == -1){
				soma += y; 
			}
		y += 1;
		}
	}

	
	printf("%i\n", soma);
	
	return 0;
}
